.github/workflows/validate.yml

```diff
@@ -24,11 +24,11 @@ jobs:
           version: v1.60
 
   codespell:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     steps:
     - uses: actions/checkout@v4
     - name: install deps
       # Version of codespell bundled with Ubuntu is way old, so use pip.
```

Comment:
Does v24.04 of Ubuntu still use a really old version of codespell?  If it has a current version, should we just switch to `apt install codespell`

Reply:
It doesn't now, but over time it will.

That reminds me of one thing I forgot here -- pin the codespell version (otherwise, when a new version is released, CI randomly breaks). Will do.

Reply:
added